README.md

```diff
@@ -109,6 +109,20 @@ env:
   value: *containerName # referencing earlier anchor in same yaml
 ```
 
+## DICOM Redactor
```

Comment:
Can you please add an entry to the table of contents?

Reply:
Ok